test/metrics/breakdown.js

```diff
@@ -0,0 +1,191 @@
+'use strict'
+
+process.env.ELASTIC_APM_TEST = true
+
+const agent = require('../..').start({
+  serviceName: 'test',
+  captureExceptions: false,
+  metricsInterval: 0
+})
+
+const http = require('http')
+const test = require('tape')
+
+const Metrics = require('../../lib/metrics')
+const mockClient = require('../_mock_http_client')
+
+const basicMetrics = [
+  'system.cpu.total.norm.pct',
+  'system.memory.total',
+  'system.memory.actual.free',
+  'system.process.cpu.total.norm.pct',
+  'system.process.cpu.system.norm.pct',
+  'system.process.cpu.user.norm.pct',
+  'system.process.memory.rss.bytes',
+  'nodejs.handles.active',
+  'nodejs.requests.active',
+  'nodejs.eventloop.delay.ns',
+  'nodejs.memory.heap.allocated.bytes',
+  'nodejs.memory.heap.used.bytes',
+  'nodejs.eventloop.delay.avg.ms'
+]
+
+if (process.platform === 'linux') {
+  basicMetrics.push('system.process.memory.size')
+}
+
```

Comment:
Are you intending to add the acceptance tests from the spec to check the values too? I think it would be a good idea to do that. IIANM, you should be able to check exact values by specifying all the start and end times.

Reply:
Working on it. Should hopefully have them up later today. 👍